app/ChatPanelHeader.tsx

```diff
@@ -7,6 +7,7 @@ import {
   Portal,
   Dialog,
   Input,
+  Text,
 } from "@chakra-ui/react";
 import {
   SidebarIcon,
```

Comment:
```suggestion
  SidebarIcon,
   ChartLineIcon,
  ListNumbersIcon,
  ChartBarIcon,
  ChartPieSliceIcon,
  PresentationChartIcon,
} from "@phosphor-icons/react";
  
  const WidgetIcons = {
  "line": <ChartLineIcon />,
  "table": <ListNumbersIcon />,
  "bar": <ChartBarIcon />,
  "pie": <ChartPieSliceIcon />,
  "insight": <PresentationChartIcon />
}
```
Mapping object for widget icons.